src/api/lib/tasks/dev.rake

```diff
@@ -99,6 +99,61 @@ namespace :dev do
       end
     end
   end
+  namespace :testdata do
+    task create: :environment do
+      unless Rails.env.to_s == 'development'
+        puts "You are running this rake task in #{Rails.env} environment."
+        puts 'Please only run this task with RAILS_ENV=development'
+        puts 'otherwise it will destroy your database data.'
+        return
+      end
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+
+      Rake::Task['db:drop'].invoke
+      Rake::Task['db:create'].invoke
+      Rake::Task['db:setup'].invoke
```

Comment:
I would consider making this independent from creating test data. Since that seems to be two different things, preparing the database and filling it with test data.

This would also allow adding these test data on top of existing ones.

Reply:
This task you can only run on a clean database otherwise it will most likely fail because some projects, attributes etc etc already exists. I wanted to avoid having everywhere if statements for checking if this object already exist. Especially for more complex factories like the maintenance project which will setup another project and also attributes etc this will be very complicated. So I think cleanest way is to just drop the db and start from scratch....